src/main/resources/templates/journeys.html

```diff
@@ -99,10 +99,15 @@ <h1 class="govuk-heading-xl ma0">Journeys for review</h1>
                     <td class="govuk-table__cell" scope="row" th:inline="text">[[${journey.volume}]]</td>
                     <td class="govuk-table__cell" scope="row" th:inline="text">
                         <span th:if="${journey.fromSiteName == null || journey.toSiteName == null}" th:inline="text">Awaiting location</span>
-                        <span th:unless="${journey.fromSiteName == null || journey.toSiteName == null}">
-                            <a th:href="@{'/add-price/' + ${journey.fromNomisAgencyId} + '-' + ${journey.toNomisAgencyId}}"
-                               class="ma0 govuk-link">Add price</a>
-                        </span>
+                        <div sec:authorize="!hasRole('ROLE_PECS_MAINTAIN_PRICE')">
```

Comment:
You will now see this on the journey for reviews page if you do not have maintain price role:

![image](https://user-images.githubusercontent.com/60104344/113684398-19c10e80-96bd-11eb-9cee-d3bec45e3caf.png)